e2e_samples/parking_sensors_synapse/synapse/notebook/02_standardize.ipynb

```diff
@@ -366,25 +186,29 @@
         "import os\r\n",
         "from opencensus.ext.azure.log_exporter import AzureLogHandler\r\n",
         "from opencensus.ext.azure.log_exporter import AzureEventHandler\r\n",
-        "#from pyspark.sql.session import SparkSession\r\n",
         "from datetime import datetime\r\n",
         "\r\n",
+        "# Getting Application Insights instrumentation key\r\n",
+        "appi_key = token_library.getSecretWithLS(keyvaultlsname,\"applicationInsightsKey\")\r\n",
+        "\r\n",
         "# Enable App Insights\r\n",
-        "aiLogger = logging.getLogger(\"ParkingSensorLogs-Standardize\")\r\n",
+        "aiLogger = logging.getLogger(__name__)\r\n",
         "aiLogger.addHandler(AzureEventHandler(connection_string = 'InstrumentationKey=' + appi_key))\r\n",
         "#logger.addHandler(AzureLogHandler(connection_string = 'InstrumentationKey=' + appi_key))\r\n",
```

Comment:
Minor cleanup, Perhaps we can remove this commented out line?

Reply:
Removed the reference to "AzureLogHandler" altogether (from both the notebooks) and added the actual queries as used in the screenshots.